src/main/java/com/dev/cinema/service/MovieSessionService.java

```diff
@@ -0,0 +1,10 @@
+package com.dev.cinema.service;
+
+import com.dev.cinema.model.MovieSession;
+import java.util.List;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    List<MovieSession> getAll();
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/03_hibernate/homework/lesson_27_1_hw.html
read task carefully
you should implement the method findAvailableSessions() on dao and service layer